sites/docs/docs/default-config.md

```diff
@@ -0,0 +1,89 @@
+---
+id: default-config-json
+title: Automatic Configuration
+sidebar_label: Automatic Configuration
+slug: /default-config-json
+---
+
+To allow quick and easy visualisation of an image or an AnnData object, Vitessce can now generate the __view config__ automatically, given the URLs to the datasets. This page explains how to use this feature and what dataset formats are supported.
+
+
+### Supported formats
+
+Vitessce currently supports automatic __view config__ generation for the following dataset formats:
+
+- [OME-TIFF](https://docs.openmicroscopy.org/ome-model/6.2.0/ome-tiff/specification.html), 
+- [OME_ZARR](https://ngff.openmicroscopy.org/latest/#on-disk),
+- Anndata-ZARR - files that have been [saved to Zarr store](https://vitessce.github.io/vitessce-python/notebooks/widget_brain.html#3.2-Save-the-Data-to-Zarr-store).
+
+
+### How to use
+
+To use this functionality, go to the [App](/#?edit=true) page of this website and paste the URL of each of the datasets that you want to visualise, separating with semicolon (`;`). Make sure that each dataset is of supported format and complies with the [requirements](/docs/default-config-json/#requirements). Vitessce will display the generated __view config__ in the editor. Then, you can either launch Vitessce with the generated configuration directly, or change it further to reflect any custom requirements.
+
+
+### Requirements
+
+You need:
+- One of more URLs, where each of them points to the static files of the dataset you want to visualise.
+- Each dataset complies with its format-specific requirements outlined below.
+
+#### OME-TIFF format
+
+If you want to view a dataset of `OME-TIFF` format, make sure that the URL to the dataset ends with `ome.tif`. For example:
+- https://example.my_dataset.ome.tif is a valid URL
+- https://example.my_dataset.ome.tiff is not valid URL
+
+#### Anndata-ZARR format
+
+If you want to view a dataset of `Anndata-ZARR` format, make sure that the URL to the dataset ends with `h5ad.zarr`, `.adata.zarr` or `.anndata.zarr`.
+
+In addition, make sure that a `.zmetadata` file, generated after running the [zarr consolidate_metadata](https://zarr.readthedocs.io/en/stable/api/convenience.html#zarr.convenience.consolidate_metadata) function, is present in the root of the dataset URL. 
```

Comment:
```suggestion
In addition, a `.zmetadata` file must be present in the root of the Zarr store. This can be generated by running the [consolidate_metadata](https://zarr.readthedocs.io/en/stable/api/convenience.html#zarr.convenience.consolidate_metadata) function on the Zarr store that results from `AnnData.write_zarr`. 
Note that we intend to [remove this requirement](https://github.com/vitessce/vitessce/issues/1459) in the future.
```